src/CornerstoneViewport/CornerstoneViewport.js

```diff
@@ -48,6 +48,7 @@ class CornerstoneViewport extends Component {
     frameRate: PropTypes.number, // Between 1 and ?
     //
     setViewportActive: PropTypes.func, // Called when viewport should be set to active?
+    setViewportSpecificData: PropTypes.func,
```

Comment:
`setViewportSpecificData` only makes sense within the context of `@ohif/viewer`. This should really be a generically named callback that makes sense within `react-cornerstone-viewport`

`onNewImage` would likely be a more appropriate name

Reply:
Code was updated with your suggestion :)